packages/core/src/routes/experience/classes/helpers.ts

```diff
@@ -129,3 +134,31 @@ export const mergeUserMfaVerifications = (
 
   return [...userMfaVerifications, ...newMfaVerifications];
 };
+
+/**
+ * Post affiliate data to the cloud service.
+ *
+ * @remarks forked from {@link routes/interaction/actions/helpers.ts}
```

Comment:
if we are going to remove the interaction directory, this link would be pointing to the void

Reply:
removed